auth/src/main/kotlin/com/alfresco/ui/SplashActivity.kt

```diff
@@ -63,16 +66,23 @@ abstract class SplashActivity : AppCompatActivity() {
     }
 
     private fun goToMain() {
-        val i = getMainIntent()
-        if (entryId.isNotEmpty()) {
-            i.flags = Intent.FLAG_ACTIVITY_REORDER_TO_FRONT
-//            i.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
-            i.putExtra(MODE_KEY, if (isPreview) VALUE_SHARE else VALUE_REMOTE)
-            i.putExtra(KEY_FOLDER, isRemoteFolder)
-            i.putExtra(ID_KEY, entryId)
+        GlobalScope.launch {
+            val checkForRoot = CheckForRootWorker(this@SplashActivity)
+            val results = checkForRoot.invoke()
+
+            if (!results.any { it.result }) {
+                val i = getMainIntent()
+                if (entryId.isNotEmpty()) {
+                    i.flags = Intent.FLAG_ACTIVITY_REORDER_TO_FRONT
```

Comment:
Variable name i can be changed

Reply:
It's previously defined variable so keeping unchanged